homeassistant/components/luftdaten/__init__.py

```diff
@@ -47,49 +50,53 @@
 
 TOPIC_UPDATE = f"{DOMAIN}_data_update"
 
-SENSORS = {
-    SENSOR_TEMPERATURE: [
-        "Temperature",
-        "mdi:thermometer",
-        TEMP_CELSIUS,
-        DEVICE_CLASS_TEMPERATURE,
-    ],
-    SENSOR_HUMIDITY: [
-        "Humidity",
-        "mdi:water-percent",
-        PERCENTAGE,
-        DEVICE_CLASS_HUMIDITY,
-    ],
-    SENSOR_PRESSURE: [
-        "Pressure",
-        "mdi:arrow-down-bold",
-        PRESSURE_HPA,
-        DEVICE_CLASS_PRESSURE,
-    ],
-    SENSOR_PRESSURE_AT_SEALEVEL: [
-        "Pressure at sealevel",
-        "mdi:download",
-        PRESSURE_HPA,
-        DEVICE_CLASS_PRESSURE,
-    ],
-    SENSOR_PM10: [
-        "PM10",
-        "mdi:thought-bubble",
-        CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
-        None,
-    ],
-    SENSOR_PM2_5: [
-        "PM2.5",
-        "mdi:thought-bubble-outline",
-        CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
-        None,
-    ],
-}
+SENSOR_TYPES: tuple[SensorEntityDescription, ...] = (
+    SensorEntityDescription(
```

Comment:
This removes the icon for this entity, we shouldn't do that in these types of PR's.
but `mdi:thermometer` is what `DEVICE_CLASS_TEMPERATURE` gives so I guess it's fine here.

Reply:
Yeah, I think you and Martin frequently mentioned that icon is redundant for `DEVICE_CLASS_TEMPERATURE`. That's the only reason I did it here (as an exception).